openedx/core/djangoapps/theming/management/commands/create_sites_and_configurations.py

```diff
@@ -75,26 +73,23 @@ def _create_oauth2_client(self, url, site_name, service_name, service_user):
             service_name=service_name,
             site_name="" if site_name == "edx" else "-{}".format(site_name)
         )
-        client, created = Client.objects.update_or_create(
+        Application.objects.update_or_create(
             client_id=client_id,
             defaults={
                 "user": service_user,
                 "name": "{site_name}_{service_name}_client".format(
```

Comment:
I think this would better match our new convention: `"{site_name}-{service_name}-sso"`.

Reply:
@dianakhuang: In Production, I don't see anything set up for discovery, but I do see the following for ecommerce (as a white label example):
- ecommerce-sso-mitxpro.mit.edu

I would hope/guess this means discovery isn't needed, but I do not know for sure.
1. The above sso config does have the `user_id` scope set up.
2. It does use "Authorization code" grant type. 
3. It looks like the naming convention would be `{service_name}-sso-{site_name}`